flixel/util/FlxTimer.hx

```diff
@@ -281,12 +309,35 @@ class FlxTimerManager extends FlxBasic
 	public function completeAll():Void
```

Comment:
Also, this doesn't really seem related to #679? Might be better to handle changes to `completeAll()` in a separate pull request.